### ISSUE TITLE
R version PDF not found on page link

### ISSUE THREAD

**Issue by avrosati** [resampling-stats/resampling-with#167]
The page link to the PDF output for the R version of the book gets a 404 (Not Found) error - did you know about this?

Thanks

T.

**Comment by matthew-brett** [comment]
Yes, sorry, there was a bug in our build configuration, fixed here : https://github.com/resampling-stats/resampling-with/commit/f1b91d15fc - the PDF should be available now.

**Comment by matthew-brett** [comment]
Can I confirm that this fixes the issue for you?  If so - would you mind closing the issue?
